massdriver-application-azure-app-service/service_principal.tf

```diff
@@ -0,0 +1,17 @@
+resource "azuread_application" "main" {
```

Comment:
this file is where we should be able to use `mdxc`. We can either set up the provider locally or I think we can cut a pre release to test the azure mdxc code path.

Reply:
```terraform
module "application" {
  source  = "github.com/massdriver-cloud/terraform-modules//massdriver-application"
  name    = var.name
  service = "function"
}
```

You'll then be able to get the service principal id with `module.application.id`.

Reply:
Nevermind, I see you have it above in `main.tf`! So we'll be able to remove most of this file. We'll want to make sure we keep that role assignment though. We can ask Chris if there's any roles being assigned in the provider.